primitives/primitives/src/traits.rs

```diff
@@ -11,6 +11,9 @@ pub trait MultiAssetRegistry<AssetId> {
     /// asset is native from the point of view of the current system
     fn native_asset_location(asset: &AssetId) -> Option<MultiLocation>;
 
+    /// Determines the chain location of the given asset
```

Comment:
I feel like this should be a bit more concise wrt `native_asset_location`.
Can we extend on this that this only returns the chain_part where as native_asset_location returns POV